stix2/core.py

```diff
@@ -21,8 +21,8 @@ def __init__(self, allow_custom=False):
     def clean(self, value):
         # Any STIX Object (SDO, SRO, or Marking Definition) can be added to
         # a bundle with no further checks.
-        if isinstance(value, (STIXDomainObject, STIXRelationshipObject,
-                              MarkingDefinition)):
+        if any(x in ('STIXDomainObject', 'STIXRelationshipObject', 'MarkingDefinition')
```

Comment:
Given that STIXDomainObject and STIXRelationshipObject are just wrappers, can we move these to a higher level and prevent the current approach? Possibly create a new one for MarkingDefinition (perhaps STIXObject?)

Reply:
Yes, we can create some base classes at the higher level. These are really just wrappers for groupings of STIX Types. The specs seem somewhat ambiguous whether MarkingDefinition (and Bundle) are STIX objects or not, so I'd rather keep those in groups by themselves.